### DISCUSSION TITLE
Phishing Abuse

### DISCUSSION THREAD

**Discussion by n-awan** [vercel/community#5437] [Help]
### Summary

Hi, a scammer is using your platform to steal Facebook credentials, I tried to email this to abuse@vercel.com but I get a bounce email saying

Message blocked
Your message to abuse@vercel.com has been blocked. See technical details below for more information.
LEARN MORE
The response was:
Message rejected. For more information, go to https://support.google.com/mail/answer/69585

### Steps to Reproduce

visit https://linkpop.com/help_business_center
proceed to the dangerous website
enter details
![unnamed](https://github.com/vercel/community/assets/148258917/fa91a3ad-f31b-42b6-9a53-3b5886323070)

![unnamed](https://github.com/vercel/community/assets/148258917/f67a998a-bfa7-4af8-a94f-4aaaf0605eb4)

![unnamed](https://github.com/vercel/community/assets/148258917/2f4a098b-9c48-45a0-b4a2-a81869867182)

![unnamed](https://github.com/vercel/community/assets/148258917/2b1823af-511b-434a-97eb-1672acf1f15d)

![unnamed](https://github.com/vercel/community/assets/148258917/1e630501-903a-40e0-8da2-195fba8e0c91)

**Comment by raj4777** [comment]
same thing with baking also now

https://sjhe7wqa.vercel.app/s1.html

![image](https://github.com/vercel/community/assets/85761926/62c8a8fc-1e32-499b-8349-1b401170b4fd)

Vercel unfortunately does not have a abuse prevention team, this is the fastest way to get themselves categorised as a spam and phishing website
